components/shutter/cdn_profile.tf

```diff
@@ -1,8 +1,8 @@
-resource "azurerm_cdn_profile" "main" {
-  name                = "hmcts-shutter-${var.env}"
-  location            = "West US"
-  resource_group_name = var.shutter_rg
-  sku                 = var.cdn_sku
+# resource "azurerm_cdn_profile" "main" {
```

Comment:
should this PR be draft? I assume this isn't ready yet?

Reply:
This should be removed entirely instead of commented out.